### PR TITLE
Add SideExtremeValue

### LINKED ISSUE #23179: Make a SideExtremeValue
## Reason
It would be helpful to be able to find the max/min value of an elemental aux variable (used for material property output). NodalExtremeValue won't work.

## Design
Make `SideExtremeValue`

## Impact
More flexibility

### PR DESCRIPTION
**Pull request by tophmatthews**
Closes #23179

### CODE REVIEW

**Comment by tophmatthews** [comment]
@dschwen can you take a look?

File: framework/src/postprocessors/SideExtremeValue.C [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      "Finds either the min or max variable value of a variable over a boundary.");
+  return params;
+}
+
+SideExtremeValue::SideExtremeValue(const InputParameters & parameters)
+  : ExtremeValueBase<SideVariablePostprocessor>(parameters),
+    _proxy_variable(isParamValid("proxy_variable") ? coupledValue("proxy_variable") : _u)
+{
+  // if (this->isNodal())
+  // this->paramError("variable", "This AuxKernel only supports Elemental fields");
```
Comment (dschwen) [inline framework/src/postprocessors/SideExtremeValue.C:32]:
```suggestion
```

**Comment by tophmatthews** [comment]
Ready for review

**Review by dschwen — APPROVED** [review]

**Comment by tophmatthews** [comment]
Thanks @dschwen !

**Pull request merged**